packages/renderer/src/lib/dashboard/ReleaseNotesBox.svelte

```diff
@@ -0,0 +1,94 @@
+<script lang="ts">
+import { faCircleArrowUp } from '@fortawesome/free-solid-svg-icons';
+import { Button, CloseButton, Link } from '@podman-desktop/ui-svelte';
+import { onDestroy, onMount } from 'svelte';
+
+import type { ReleaseNotes } from '/@api/release-notes-info';
+
+import Markdown from '../markdown/Markdown.svelte';
+
+let showBanner = false;
+let notesAvailable = false;
+let updateAvilable = false;
+let notesURL: string;
+let currentVersion: string;
+let notesInfo: ReleaseNotes | undefined;
+const receiveShowReleaseNotes = window.events?.receive('show-release-notes', () => {
+  showBanner = true;
+});
+function openReleaseNotes() {
+  window.openExternal(notesURL);
+}
+function updatePodmanDesktop() {
+  window.updatePodmanDesktop();
+}
+async function getInfoFromNotes() {
+  let getNotes = await window.podmanDesktopGetReleaseNotes();
+  notesAvailable = getNotes.releaseNotesAvailable;
+  if (notesAvailable) {
+    notesInfo = getNotes.notes;
+  }
```

Comment:
what happens if we use directly
```suggestion
    notesInfo = releaseNotes?.notes;
```